build/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1.405-bionic AS build
+FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
```

Comment:
@brendankowitz I have updated the .net sdk image name from the [docker hub documentation](https://hub.docker.com/_/microsoft-dotnet-sdk/). I am not sure if we should find the bionic version of 5.0.100 as I couldn't find any documentation around this. Thoughts?